<launch>
  <arg name="camera" default="camera"/>
  <arg name="tag_family" default="tf36h11"/>
  <node pkg="apriltag_detector" type="apriltag_detector_node"
	ns="$(arg camera)"
	name="apriltag_detector" clear_params="true"
	output="screen">
    <param name="tag_family" value="$(arg tag_family)"/>
    <remap from="~image" to="image_raw"/>
  </node>
</launch>
